src/services/compare/handlers/sendDoesNotExistAlert.js

```diff
@@ -33,25 +40,115 @@ exports.handler = async function (event, context, callback) {
         message: `Alert for ${data.id} - TEST `,
       });
     } else {
-      const { emailRecipients, sourceEmail } = await getSecretsValue({
+      let isProgramTypeChp = false;
```

Comment:
im confused as to why you are defining this as false and then changing the value. we can just define it at the top level as `const isCHP = data.programType === "CHP"` instead of redefining